src/engine/Sound.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <unordered_map>
+#include "Buffer.h"
+
+
+class Sound
```

Comment:
This class is doing both what SoundComponent and SoundSystem would supposedly be doing.